test/tools/libtesteth/TestHelper.cpp

```diff
@@ -499,8 +532,9 @@ void addClientInfo(json_spirit::mValue& _v, fs::path const& _testSource)
 				comment = existingInfo["comment"].get_str();
 		}
 
+		string const& source = _testSource.string();
 		clientinfo["filledwith"] = prepareVersionString();
-		clientinfo["source"] = _testSource.string();
+		clientinfo["source"] = source.substr(source.rfind("/src/"));
```

Comment:
I oppose parsing and decomposing strings here.

Reply:
what your suggestion then?

Reply:
`addClientInfo()` should not receive the absolute path.  It should receive the path to be printed.